frontend/packages/ux-editor/src/hooks/useSelectedFormLayoutSetName.ts

```diff
@@ -1,22 +1,25 @@
 import { useStudioUrlParams } from 'app-shared/hooks/useStudioUrlParams';
 import { useLayoutSetsQuery } from 'app-shared/hooks/queries/useLayoutSetsQuery';
-import { useReactiveLocalStorage } from 'app-shared/hooks/useReactiveLocalStorage';
+import { useEffect, useState } from 'react';
 
```

Comment:
I suggest that we add the  `storageKey` within a variable. 😊 
```suggestion
const storageKey: string = 'selectedFormLayoutSetName';
```

Reply:
fixed